src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write your code here
+const population = [...document.querySelectorAll('.population')];
+const toNumbers = population.map((element) => {
+  return parseInt(element.textContent.replace(/,/g, ''));
+});
+
+const totalSum = toNumbers.reduce((sum, element) => sum + element, 0);
+const average = Math.round(totalSum / toNumbers.length);
+
+document.querySelector('.average-population').textContent =
+  average.toLocaleString();
+
+document.querySelector('.total-population').textContent =
+  totalSum.toLocaleString();
```

Comment:
The code is repeatedly querying the DOM to find the '.average-population' and '.total-population' elements. This can be inefficient, especially if the DOM is large or if this operation is done frequently. Consider storing these elements in variables at the start of the script and reusing those variables.